### PR TITLE
Markdown & Project links

### PR DESCRIPTION
**Pull request by pablo-mayrgundter**
Hi Oleg,

Here's a first cut at Markdown support (#273).  Posting as a draft to validate against live github issues on our home project.

I would appreciate any pointers you can give me for where an embedded URL is being handled such that clicking on it doesn't lead to the full project path.

### CODE REVIEW

**Comment by pablo-mayrgundter** [comment]
Note the italics in the comment.. these are driven by markdown.
image

**Comment by pablo-mayrgundter** [comment]
I've also setup a test issue that demonstrates a link to another project (KNIK):

https://deploy-preview-280--bldrs-share.netlify.app/share/v/gh/pablo-mayrgundter/ifctool/main/index.ifc#i:1317978906

See the third comment.  You can click on KNIK.  Link isn't styled.. working on that now.

**Comment by OlegMoshkovich** [comment]
@pablo-mayrgundter
We extract url on line 195 here:
https://github.com/bldrs-ai/Share/blob/main/src/Components/IssuesControl.jsx

the url is used for the camera view on line 75 and 89 here:
https://github.com/bldrs-ai/Share/blob/main/src/Components/IssueCard.jsx

File: src/Components/IssueCard.jsx [hunk 1]
```diff
@@ -311,13 +311,18 @@ const useStyles = makeStyles((theme) => ({
...
   },
   body: {
-    height: (props) => props.expandText ? 'auto' : '62px',
-    margin: '5px',
-    paddingLeft: '5px',
-    overflow: 'hidden',
-    textOverflow: 'ellipsis',
-    fontSize: '1em',
-    lineHeight: '1.3em',
+    'height': (props) => props.expandText ? 'auto' : '62px',
```
Comment (OlegMoshkovich) [inline src/Components/IssueCard.jsx:314]: image

something has shifted with the markdown perhaps adjust 62px to 58px -- perhaps?

**Review by OlegMoshkovich — COMMENTED** [review]
one minor comment.

File: src/Components/IssueCard.jsx (hunk 1 above)
Comment (pablo-mayrgundter) [reply src/Components/IssueCard.jsx:314]: Done

**Review by OlegMoshkovich — APPROVED** [review]

**Pull request merged**